test/test_browser.py

```diff
@@ -5887,3 +5887,12 @@ def setUp(self):
     self.set_setting('MEMORY64')
     self.emcc_args.append('-Wno-experimental')
     self.require_wasm64()
+
+class browser64_4gb(browser):
+  def setUp(self):
+    super().setUp()
+    self.set_setting('MEMORY64')
```

Comment:
Separate from this PR, could `require_wasm64()` set `MEMORY64` to avoid the duplication?